tests/conftest.py

```diff
@@ -9,24 +14,51 @@ def autouse_django_db(db):
     pass
 
 
-@pytest.fixture
-def notification_template(settings):
+@pytest.fixture(autouse=True)
+def force_settings(settings):
     settings.LANGUAGES = (("fi", "Finnish"), ("en", "English"))
-    notifications.register("event_created", "Event created")
+    settings.LANGUAGE_CODE = "en"
+
+
+@pytest.fixture
+def notification_template():
+    notification_type = "event_created"
```

Comment:
Not sure the purpose of assigning "event_created" to a variable. I couldn't see any other use of `notification_type` or am I missing anything here

Reply:
Just so that the places where `"event_created"` is required, a single value coming from the variable is used instead of having it hard-coded in different places.

Reply:
now that it used a few lines below, makes at least some sense to have it in a var 😅  when the var was not used at all, it was a bit silly indeed 😬